website/content/partials/components/configsourcer-vault.mdx

```diff
@@ -82,7 +82,7 @@ If this is not set, Vault agent will not be used. This should only be set if you
 
 The authentication method to use for Vault.
 
-This can be one of: `aws`, `kubernetes`. When this is set, configuration fields prefixed with the auth method type should be set, if required. Configuration fields prefixed with non-matching auth method types will be ignored (except for type validation). If no auth method is set, Vault assumes proper environment variables are set for Vault to find and connect to the Vault server. When this is set, `auth_method_mount_path` is required.
+This can be one of: `aws`, `kubernetes`. When this is set, configuration fields prefixed with the auth method type should be set, if required. Configuration fields prefixed with non-matching auth method types will be ignored (except for type validation).  If no auth method is set, Vault assumes proper environment variables are set for Vault to find and connect to the Vault server. When this is set, `auth_method_mount_path` is required.
```

Comment:
Hrm, not sure why this has an extra space? 🤔 Maybe the new line characters are ignored?

https://github.com/hashicorp/waypoint/blob/99d1d67aca31349d4edf84198916e6769ed996aa/builtin/vault/config_sourcer.go#L451-L459

Reply:
Hm, I saw that too. I'll check it out.